content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,294 @@
+
+For more details and other ways of registering model versions you can have a
+look at the corresponding
+[documentation](/doc/studio/user-guide/model-registry/register-version).
+
+Once we register our first model version, DVC Studio will also automatically
+connect it to experiment tracking and all metrics which are tracked there will
+also show up in the model registry for each model version.
+
+<details>
+
+#### 💡 Expand to see how registering models works under the hood
+
+Registering the model version as we just did using DVC Studio is equivalent to
+the following GTO command
+
+```
+gto register pool-segmentation [ref] --version v1.0.0
+```
+
+Here, `[ref]` is the Git reference/hash we selected from the menu in DVC Studio.
+
+For more details you can have a look at the
+[gto register command reference](/doc/gto/command-reference/register).
+
+</details>
+
+## Assigning lifecycle stages
+
+We have a first version for our model and now it is a good time to assign a
+model lifecycle stage to it. You can create any number of lifecycle stages with
+any names you wish but in this example we will only create two stages called
+"dev" and "prod".
+
+Stages are created whenever a model version is assigned to them. You can now
+assign the model version 1.0.0 to the "dev" stage as follows.
+
+![Assigning model stages](/img/mr-assign-model-stage.gif)
+
+When we assing the model to a stage, it can automatically trigger actions in our
+CICD workflows, like deploying the model to a new environment (we will explore
+how this is done in the
+[Using and Deploying models](/doc/start/model-management/model-cicd) chapter).
+
+<details id="under-the-hood-assigning-model-stages">
+
+#### 💡 Expand to see how assigning model stages works under the hood
+
+Assigning the "dev" stage to the model as we just did using DVC Studio is
+equivalent to the following GTO command.
+
+```
+gto assign pool-segmentation --version v1.0.0 --stage dev
+```
+
+For more details you can have a look at the
+[gto assign command reference](/doc/gto/command-reference/assign).
+
+</details>
+
+## Changing (and removing) stage assignments
+
+Let's say that we've decided to promote our model version 1.0.0 to production
+and denote that it is no longer in the "dev" stage. First, assign the model
+version to the "prod" stage just like we did with the "dev" stage in the
+previous section.
+
+Now, to remove the "dev" stage from our model version 1.0.0 and it assigned only
+to "prod", follow these steps:
+
+![Un-assigning model stages](/img/mr-unassign-model-stage.gif)
+
+It is also possible to de-register model versions or deprecate and remove models
+from the registry entirely. To see how, have a look at the
+[documentation](/doc/studio/user-guide/model-registry/remove-a-model-or-its-details).
+
+The detailed view of our model in the registry should now match what we see
+[in our example](<(https://studio.iterative.ai/team/Iterative/models/b3P4bcYIrGYdzyjqzsf9Xw==/pool-segmentation/v1.0.0)>).
```

Comment:
Again this gives a 404

Reply:
Same as above

Reply:
Before we merge we should check that this works after merging [this](https://github.com/iterative/example-repos-dev/pull/271). Maybe the link will change

Reply:
Captured in #4990